### ISSUE TITLE
Setup Travis CI

### ISSUE THREAD

**Issue by Nava2** [umple/umple.gradle#4]
Setup this project to build under Travis CI.

All that should be performed in Travis: 

1. `./gradlew check`

It's not a lot, but it gives a point of knowledge for failure. It looks like the `check` task is run by default, this setup should be quick and easy. 👍

**Comment by Nava2** [comment]
The work in #9 will let this become a reality. Please do this after the base for #9 is complete, @AdamBJ.

**Comment by AdamBJ** [comment]
I've started on this. #12 is me trying to get everything working. The problem right now is that Travis doesn't have access to the local `umple-latest` jar. One workaround that I can think of is to change the `download` task to this:

```
download {
    onlyIfNewer = false
    overwrite = true
    src 'http://cruise.eecs.uottawa.ca/umpleonline/scripts/umple.jar'
    dest 'libs/umple-latest.jar'
} 
```

Rather than only downloading if the copy on the server is newer, we always download it.

**Comment by AdamBJ** [comment]
This was closed by PR#12
